platform/client/src/context/AuthContext.tsx

```diff
@@ -18,6 +19,13 @@ export const AuthContext = React.createContext<IAuthContext>({
 export const AuthContextProvider = (props: any) => {
     const setVolunteer = (volunteer: VolunteerType) => {
       setState({...state, volunteer: volunteer})
+      if (volunteer?._id) {
+        const filter = {
+          volunteerId: { eq: volunteer._id }
+        };
+        VolunteerActionModel.applyFilter(filter);
+        datastore.startReplication();
```

Comment:
Love it. This could land in the docs as well.
Really good job

Reply:
Awesome, yeah I can add this to the docs :)